### PR TITLE
Mr/documentation changes

### PR DESCRIPTION
**Pull request by marothe**
Only a handful of spelling changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Scanbot Barcode Scanner SDK Xamarin Examples
+## Scanbot Barcode Scanner SDK Xamarin Examples
```
Comment (Nikituh) [inline README.md:1]: I'm pretty sure we should keep the single #hashtag here, it's the *heading* of the file. 

Pics:
Screenshot 2021-03-03 at 15 32 28

Screenshot 2021-03-03 at 15 32 37

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ For more details about the Scanbot Barcode Scanner SDK for Xamarin please see th
 
 The Scanbot Barcode Scanner SDK will run without a license for one minute per session!
 
-After the trial period is over all SDK functions as well as the UI components (like Barcode Scanner UI) will stop working.
+After the trial period has expired all SDK functions as well as the UI components (like Barcode Scanner UI) will stop working.
 You have to restart the app to get another trial period.
 
-To get an unrestricted "no-strings-attached" 30 day trial license, please submit the [Trial License Form](https://scanbot.io/en/sdk/demo/trial) on our website.
+To get an unrestricted, "no-strings-attached" 30-day trial license, please submit the [Trial License Form](https://scanbot.io/en/sdk/demo/trial) on our website.
```
Comment (Nikituh) [inline README.md:32]: I would say `unrestricted` and `trial` here are cumulative adjectives and do not require a comma.

https://prowritingaid.com/grammar/1008103/Should-I-use-a-comma-between-two-adjectives

Comment (marothe) [reply README.md:32]: @cheetoblue Would you mind chiming in here? Relying on your native expertise here :-).

File: README.md (hunk 1 above)
Comment (marothe) [reply README.md:1]: My mistake, sorry! Did not realize that using my IDE, thought it was style-only.

File: README.md (hunk 2 above)
Comment (cheetoblue) [reply README.md:32]: They way I see it, "unrestricted" means that you can access all the features of the app and "no-strings-attached" means that there is no contract binding you to purchasing the app. These are thus two separate adjectives explaining two separate parts of the trial license. As for the "30 day trial license" - in theory "30 day" is also an adjective explaining the trial license, but "30 day trial license" has become so common that it is actually seen as a single entity (and thus a noun, if I can use poetic licensing here) :)

**Pull request merged**